build.go

```diff
@@ -268,6 +268,30 @@ func runComposerInstall(
 				logger.Debug.Subprocess(fmt.Sprintf("- %s", f.Name()))
 			}
 		}
+		if os.Getenv("BP_RUN_COMPOSER_INSTALL") == "1" {
```

Comment:
If we do end up going with the environment variable, we usually use booleans for this type of environment variable, so I'd want to see this changed to true/false. I'd also want to see this env. var documented in the README with the rest of the configurations. 

Let's not make these changes until we (you and @paketo-buildpacks/php-maintainers) are in agreement about how to do this